pkg/policies/branch/branch.go

```diff
@@ -397,6 +462,35 @@ func fix(ctx context.Context, rep repositories, c *github.Client,
 				update = true
 			}
 		}
+		if pr.RequiredStatusChecks == nil && (mc.RequireUpToDateBranch || len(mc.RequireStatusChecks) > 0) {
+			rsc := &github.RequiredStatusChecks{
+				Strict: mc.RequireUpToDateBranch,
+				// Can't be set to nil, so force non-nil value.
+				Contexts: append([]string{}, mc.RequireStatusChecks...),
+			}
+			pr.RequiredStatusChecks = rsc
+			update = true
+		} else if mc.RequireUpToDateBranch || len(mc.RequireStatusChecks) > 0 {
+			if mc.RequireUpToDateBranch && !pr.RequiredStatusChecks.Strict {
+				pr.RequiredStatusChecks.Strict = true
+				update = true
+			}
+			if len(mc.RequireStatusChecks) > 0 {
+				allContexts := make(map[string]struct{}, len(pr.RequiredStatusChecks.Contexts))
+				for _, ctx := range pr.RequiredStatusChecks.Contexts {
+					allContexts[ctx] = struct{}{}
+				}
+				for _, ctx := range mc.RequireStatusChecks {
+					allContexts[ctx] = struct{}{}
+				}
+				pr.RequiredStatusChecks.Contexts = make([]string, 0)
```

Comment:
Again, should use Checks.Context.
https://docs.github.com/en/rest/reference/branches#update-status-check-protection
Looks like we can null out the AppID.

Reply:
Updated in https://github.com/ossf/allstar/commit/a57c5c2d9b7c76ec7af91d9dd5375b39f3c090b1